### ISSUE TITLE
Need the reference hyper-parameters and output of the default scripts.

### ISSUE THREAD

**Issue by jeffchy** [hpcaitech/Open-Sora#47]
I ran the training scripts (using DiT-S/8 by default) successfully, and the loss curve is shown below.
![image](https://github.com/hpcaitech/Open-Sora/assets/17855610/99a23790-a9c2-4736-95b9-c842176dea2a)
However, the sampled results (also using the default sampling parameters with DiT-XL/2 modified to DiT-S/8） are random noises.

Is that because the model is weak?
Could you please provide a recommended setting (hyper-params like the model arch, compression, etc) that we should start with?

**Comment by zhengzangw** [comment]
Your number of training steps is quite small. We have included recommended training setting in `config`, and training loss in our report.
